src/index.css

```diff
@@ -92,7 +92,7 @@ hr {
   color: #afafaf;
 }
 
-.button:hover:not(:disabled) {
+.button:hover:not(:disabled), .button:focus:not(:disabled) {
```

Comment:
Guess this would apply to all buttons with `.button` class. Not sure if this would be a problem for other buttons in the app.

Reply:
Taking a closer look and thinking that maybe I can relocate this to be more less global. Let me know what you think!